scripts/integration_test_contracts.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash -e
+
+# sleep
+# This sleep is the sum of the time when docker is pulled, started,
+# and the height of the node increases to 1 or more.
+sleep 30
```

Comment:
could you use other than simple sleep? For example, check the block height and retry after some sleep.

Reply:
Required as time to download docker.

Reply:
Fixed to wait until height becomes 1 or more

[fix: correct to wait for processing until height becomes 1 or more](https://github.com/Finschia/finschia-wasm/pull/3/commits/105bc0da23b62dd11851c972792742a88e4d8476)